sherpa/astro/tests/test_astro_data2.py

```diff
@@ -3773,11 +3957,11 @@ def test_pha_subtract_bkg_filter_false():
     data.set_background(bkg)
     data.subtract()
 
-    bgot = bkg.get_staterror(filter=False, staterrfunc=lambda x: np.sqrt(x))
+    bgot = bkg.get_staterror(filter=False, staterrfunc=np.sqrt)
```

Comment:
Should we test that the old lambda case also works? In this case, `lambda x: np.sqrt(x)` is of course identical to `np.sqrt`, but maybe a user wants something different like `lambda x: np.sqrt(x + 1)`. Not necessarily in scope for this test, but I wonder if that was the motivation behind using the lambda here originally.

Reply:
This was a pylint-inspired change. Given the mechanics of function application I don't see the need for something clever, since "lambda x: np.sqrt(x)" and "np.sqrt" are essentially the same. I don't see the need to add something extra.